ts/src/base/errorHierarchy.ts

```diff
@@ -45,8 +46,8 @@ const errorHierarchy = {
                 'InvalidNonce': {},
                 'RequestTimeout': {},
             },
+            'OrderBookNonceMismatch': {},
```

Comment:
@ttodua, should we update this exception?

Reply:
ah, seems I had submitted it there. plz feel free to name it whichever you like